src/components/StaticContent/HowItWorks.re

```diff
@@ -53,11 +53,11 @@ module HowItWorks = {
                   <hr />
                   <Text>
                     <S>
-                      "You can become the patron of Vitalik the animal by purchasing him for his current sale price. Vitalik the animal is "
+                      "You can become the guardian of a wildcard by simply clicking buy on any of the listed wildcards. Wildcards are "
```

Comment:
This is good. I've been thinking of changing the word "Buy", to "Guardian" on the buttons. But buy is much more clear.